Loading checkpoint shards: 100%|███████████████████████████████████████████████████████████████████████| 8/8 [00:16<00:00,  2.02s/it]
INFO:__main__:Tokenizer max length: 1000000000000000019884624838656
INFO:__main__:Loading datasets...
Map: 100%|█████████████████████████████████████████████████████████████████████████████| 1319/1319 [00:00<00:00, 19714.30 examples/s]
Traceback (most recent call last):
  File "/home/llm4math/LLM-for-Math/Direct Verifier/code/train.py", line 82, in <module>
    main(args)
  File "/home/llm4math/LLM-for-Math/Direct Verifier/code/train.py", line 37, in main
    train_dataset, val_dataset = load_train_val_datasets(
                                 ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/llm4math/LLM-for-Math/Direct Verifier/code/data/dataset_loader.py", line 50, in load_train_val_datasets
    generate_data = Dataset.from_dict(generate_data).shuffle(seed=42)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/llm4math/miniconda3/envs/mathllm/lib/python3.12/site-packages/datasets/arrow_dataset.py", line 920, in from_dict
    for col, data in mapping.items():
                     ^^^^^^^^^^^^^
AttributeError: 'Dataset' object has no attribute 'items'. Did you mean: 'iter'?
[rank1]: Traceback (most recent call last):
[rank1]:   File "/home/llm4math/LLM-for-Math/Direct Verifier/code/train.py", line 82, in <module>
[rank1]:     main(args)
[rank1]:   File "/home/llm4math/LLM-for-Math/Direct Verifier/code/train.py", line 37, in main
[rank1]:     train_dataset, val_dataset = load_train_val_datasets(
[rank1]:                                  ^^^^^^^^^^^^^^^^^^^^^^^^
[rank1]:   File "/home/llm4math/LLM-for-Math/Direct Verifier/code/data/dataset_loader.py", line 50, in load_train_val_datasets
[rank1]:     generate_data = Dataset.from_dict(generate_data).shuffle(seed=42)
[rank1]:                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[rank1]:   File "/home/llm4math/miniconda3/envs/mathllm/lib/python3.12/site-packages/datasets/arrow_dataset.py", line 920, in from_dict
[rank1]:     for col, data in mapping.items():
[rank1]:                      ^^^^^^^^^^^^^
[rank1]: AttributeError: 'Dataset' object has no attribute 'items'. Did you mean: 'iter'?
